app/views/layouts/_review_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_for [@park, @review] do |f| %>
```

Comment:
This form should live in `app/views/reviews` and it should be named `_form.html.erb`